index.html

```diff
@@ -197,33 +205,40 @@ <h3 class="multi">Skills</h3>
                         </ul>
                     </article>
                 </div>
-                <!-- under work for responsive -->
             </section>
+
             <div>
                 <h1 id="contact"></h1>
             </div>
             <section class="endp" id="#contact">
                 <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
                 </p>
                 <section class="contact_form">
-                    <form action="https://formspree.io/f/myyvrwdd" method="POST"><label for="first_name"></label><input
-                            name="first_name" type="text" required placeholder="Full Name" maxlength="30"
-                            minlength="2"><br><label for="email"></label><input name="email" type="email" required
-                            placeholder="Email Address"><label for="message"></label><textarea name="message" cols="30"
-                            rows="5" placeholder="Enter your message here ..." required maxlength="500"
-                            minlength="2"></textarea><br><button type="submit" class="getintouch">Get in touch</button>
-                    </form>
-                </section>
+                <form novalidate action="https://formspree.io/f/myyvrwdd" method="POST"><label for="first_name"></label><input
```

Comment:
- [ ] Currently, your form can be submitted with no data. The `novalidate` attribute is preventing the `required` attribute from the inputs to prevent the user to submit the form with all the input fields filled. Kindly remove this attribute so your form can perform as expected.